streamparse/util.py

```diff
@@ -120,7 +120,10 @@ def activate_env(env_name=None):
     """
     env_name, env_config = get_env_config(env_name)
 
-    env.storm_workers = get_storm_workers(env_config)
+    if options and 'storm.workers.list' in options:
```

Comment:
Is there a case where we'd want to keep an empty list here? Seems like we'd be better off with

```py
if options.get('storm.workers.list'):
```